### DISCUSSION TITLE
文字转语音的时候，在聊天页面的翻译比较准，然后进入“跟读训练”后，句子跟读的会出现偏差

### DISCUSSION THREAD

**Discussion by JKaren24** [ZuodaoTech/everyone-can-use-english#400] [General]
![1](https://github.com/xiaolai/everyone-can-use-english/assets/25974167/b4cd8f76-a230-4334-b7b4-021f55f33e63)
![Xnip2024-03-09_19-58-11](https://github.com/xiaolai/everyone-can-use-english/assets/25974167/df149649-6114-4afb-85da-24b0780bab6a)

**Comment by JKaren24** [comment]
enjoy版本 13

**Comment by an-lee** [comment]
嗯。我想办法优化一下
